libnopegl/src/node_drawother.c

```diff
@@ -399,6 +412,23 @@ static const struct node_param drawhistogram_params[] = {
 };
 #undef OFFSET
 
+#define OFFSET(x) offsetof(struct drawmask_opts, x)
+static const struct node_param drawmask_params[] = {
+    {"content",   NGLI_PARAM_TYPE_NODE, OFFSET(content),
+                 .flags=NGLI_PARAM_FLAG_NON_NULL,
+                 .node_types=(const uint32_t[]){TRANSFORM_TYPES_ARGS, NGL_NODE_TEXTURE2D, NGLI_NODE_NONE},
+                 .desc=NGLI_DOCSTRING("texture being masked")},
```

Comment:
if it's referenced as a texture in the doc, maybe change the field name to texture ? (or the other way around)